Solutions/Recorded Future/Playbooks/RecordedFuture-IOC_Enrichment(IP_Domain_URL_Hash).json

```diff
@@ -0,0 +1,543 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"metadata": {
+		"comment": "This playbook leverages the Recorded Future API to automatically enrich the IP, Domain, Url & Hash indicators, found in Sentinel incidents, with the following context: Risk Score, Risk Rules and Link to Intelligence Card. The enrichment content will be posted as a comment in the Sentinel incident",
+		"author": "Adrian Porcescu, Recorded Future"
+	},
+	"parameters": {
+		"PlaybookName": {
+			"defaultValue": "RecordedFuture-IOC_Enrichment(IP_Domain_URL_Hash)",
+			"type": "string"
+		}
+	},
+	"variables": {
+		"RecordedFutureConnectionName": "[concat('recordedfuture-', parameters('PlaybookName'))]",
```

Comment:
Please remove the ( ) from name. This is not a valid format during deployment. Because of it deployment fails.